src/main/java/org/takes/rs/RsVelocity.java

```diff
@@ -189,4 +205,31 @@ public Pair(final CharSequence key, final Object obj) {
             super(key, obj);
         }
     }
+
+    /**
+     * Template folder.
+     */
+    public static final class TemplateFolder {
```

Comment:
@ptrecenti I didn't get the idea here, why do we need this class? Can't we just use `String` or `Path` or `File`?